charts/llmos-agents/templates/langflow/frontend-deployment.yaml

```
@@ -0,0 +1,111 @@
+    {{- include "llmos-agents.langflow.labels" . | nindent 4 }}
+    langflow-scope: "frontend"
+spec:
+  replicas: {{ .Values.langflow.frontend.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "llmos-agents.langflow.selectorLabels" . | nindent 6 }}
+      langflow-scope: "frontend"
+  template:
+    metadata:
+      {{- with .Values.langflow.frontend.podAnnotations }}
+      annotations:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)